src/app/page.tsx

```
@@ -188,151 +188,73 @@ const Home = () => {
               <p className="text-gray-400 mt-4">
-                We’ve made customization easy and intuitive, so you don’t need coding skills to make it happen.
-                Simply configure and deploy, adjusting in real time to match your needs.
+                Adjust and view your customized setup in real-time without ever needing to code.
               </p>
             </div>
           </div>
-          <button className="bg-[#e74c3c] text-white px-8 py-4 rounded-md font-medium hover:bg-[#c0392b] transition-colors">
+          <button
+            className="mt-8 bg-[#e74c3c] text-white px-10 py-4 rounded-md font-medium hover:bg-[#c0392b] transition-colors"
+            onClick={() => window.location.href = '/download'}
+          >
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)